tools/abel/README.md

```diff
@@ -1,191 +1,233 @@
 
-### Update Workflow
 
-If you would like to create a building configuration to update an already onboarded building, there are two options:
+3. Select option 1, edit or update an existing building config, by entering a “1” into the command prompt.This indicates that you do not have an existing spreadsheet and would like to create one off of an initial building configuration file. Note that the building configuration file’s metadata is set to update or initialize for this step to be successful.
 
-Generate an ABEL spreadsheet from an exported building configuration.
 
-  1. In `digitalbuildings/tools/abel` run ABEL with the command:
+4. Make any edits or additions to the building entities using the ABEL spreadsheet generated.
 
-      ```
-      python3 abel.py -c /path/to/credential.json -b absolute/path/to/building/config
-      ```
-To generate a building config from an already-updated spreadsheet.
 
-  1. In `digitalbuildings/tools/abel` run ABEL with the command:
+5. Return to the command prompt when finished and press any key to perform validation and generate a new building configuration YAML file.
 
-      ```
-      python3 abel.py -s <input_spreadsheet_id> -c <path/to/client_credential.json> -d <path/to/output/directory>
-      ```
 
-  3. If your spreadsheet does not pass the validation criteria found in the
-    [spreadsheet docs](../../tools/abel/validators/README.md) then ABEL will fast-fail and a validation report will be created in the current directory with the name `spreadsheet_validation_<todays_date_and_time>.log`
+#### Generate a Building Configuration File from an Existing Spreadsheet
 
-  4. The resulting building configuration file and instance validation report will be written to the current directory with the following names:
-      * `bc_export_<today_date_and_time>.yaml`
-      * `instance_validation_<today_date_and_time>.log`
+Use the commands below if you have already created and edited an ABEL spreadsheet for the entities you would like to onboard. This process will convert your edited spreadsheet into a building configuration YAML file. 
 
-### Initialization Workflow
-If you would like create a building configuration file under the initialization operation
+1. Run the ABEL command with the arguments provided. Note that the arguments in brackets are optional. See the [ABEL Arguments](#abel-arguments) section for details.
+```
+python3 abel.py -s <input_spreadsheet_id>  -c </path/to/credential.json>
+[-p  <projects/google.com:your-project/subscriptions/your-subscription>] [-o <300>] [-m </path/to/ontology>]
+[-d </path/to/desired/output/directory>]
+```
 
-The process for using an ABEL spreadsheet to generate a new building configuration is as
-follows:
+2. Select option 1, modify a spreadsheet/building config for an existing building, by entering a “1” into the command prompt. This indicates that you are working with an already onboarded building.
```

Comment:
nit: suggest formatting the "option 1, modify a spreadsheet..." section as code with backticks so it stands out against the rest of the text